packages/server/graphql/mutations/addFeatureFlag.ts

```diff
@@ -34,14 +35,26 @@ export default {
   },
   async resolve(
     _source: unknown,
-    {emails, domain, flag}: {emails: string[] | null; domain: string | null; flag: string},
+    {
+      emails,
+      domain,
+      flag
+    }: {emails: string[] | null; domain: string | null; flag: UserFeatureFlagEnum},
     {authToken, dataLoader}: GQLContext
   ) {
     const operationId = dataLoader.share()
     const subOptions = {operationId}
 
     // AUTH
-    requireSU(authToken)
+    if (!isSuperUser(authToken)) {
```

Comment:
We want users to grant themselves access to the Spotlight feature by typing in the cheat code. The other flags, poker and jira, aren't currently being used so we don't need to require super user access here. 

As we add more feature flags, I think it's unlikely that we would want to prevent users from adding themselves to a feature. If they're so excited by a feature that they read the codebase and add themself, that's probably fine. If we create a feature that only paying users can access, we can then refactor this logic or create a private schema mutation.

Reply:
I agree, I don't see any risk in users enabling beta features for themselves.

Reply:
I like it! Super users could still add the feature flag for other emails & domains in my previous implementation, but not including `email` when adding the flag for the viewer is much cleaner. Updated!